apps/google-analytics-4/frontend/src/components/config-screen/api-access/service-account/SetupServiceAccountCard.spec.tsx

```diff
@@ -0,0 +1,21 @@
+import SetupServiceAccountCard from "./SetupServiceAccountCard";
+import { act, render, screen } from '@testing-library/react';
+import { validServiceKeyFile } from "../../../../../test/mocks";
```

Comment:
this is perhaps out of scope for this PR but figured it might be worth mentioning for the future. I have had a few tough experiences with file restructuring because of relative imports. What are any thoughts on having non-relative imports instead?

Reply:
Fan of this idea as well